test/replication/qsync_advanced.result

```diff
@@ -169,6 +175,9 @@ box.space.sync:select{} -- 1, 2, 3
  |   - [2]
  |   - [3]
  | ...
+test_run:wait_lsn('replica', 'default')
```

Comment:
Here as well the insertions are done with quorum 2, so there's no need to wait for lsn on replica